x-pack/solutions/observability/plugins/slo/public/pages/slo_details/components/external_link_display_text.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiLink, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+export function ExternalLinkDisplayText({
+  textSize,
+  id,
+  url,
+}: {
+  textSize: 's' | 'xs';
+  id: string;
+  url: string;
+}) {
+  return (
+    <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+      <EuiFlexItem grow={false}>
+        <EuiText size={textSize}>{id}</EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiLink data-test-subj="sloHealthCalloutInspectTransformLink" href={url}>
```

Comment:
This is used now to inspect the SLO itself AND the transform.
I'm wondering if we should change the `data-test-subj`.

```suggestion
        <EuiLink data-test-subj="sloHealthCalloutInspectLink" href={url}>
```